server/views/pages/bookingJourney/selectAdditionalSupport.njk

```diff
@@ -0,0 +1,81 @@
+{% extends "../../partials/layout.njk" %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set pageTitle = "Any additional support needs?" %}
+
+{% set backLinkHref = "/" %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {% include "partials/errorSummary.njk" %}
+
+      <h1 class="govuk-heading-l">Is additional support needed for any of the visitors?</h1>
+
+      <p>Tell us about any support that visitors will need because of a health condition of disability. For example, wheelchair access.</p>
+      <p data-test="prison-name">{{ bookingJourney.prison.prisonName }} will make reasonable adjustments for the visit, but may not be able to fulfil your request</p>
+
+      <form action="/book-a-visit/select-additional-support" method="POST" novalidate>
+        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+        {% set otherSupportHtml %}
```

Comment:
might be better named something like `supportDetailsHtml` - I think the `'other'` is a hangover from when this field was an 'Other' option in Staff service.